keyboards/numatreus/keymaps/hdbx/keymap.c

```diff
@@ -0,0 +1,237 @@
+   * |   Q  |  Up  |   E  |   R  |   T  |             |   7  |   8  |   9  |   .  | Bspc |
+   * |------+------+------+------+------|             |------+------+------+------+------|
+   * | Left | Down |Right |   F  |   G  |             |   4  |   5  |   6  |   +  |   *  |
+   * |------+------+------+------+------|             |------+------+------+------+------|
+   * |   Z  |   X  |   C  |   V  |   B  |             |   1  |   2  |   3  |   -  |   /  |
+   * |------+------+------+------+------+-------------+------+------+------+------+------|
+   * | Alt  | Esc  |  Tab |Shift |Space | Ctrl |toBase|   \  |   0  |   ,  |   =  |Enter |
+   * `-----------------------------------------------------------------------------------'
+   */
+  [_GAME] = LAYOUT( \
+      KC_Q,    KC_UP,   KC_E,    KC_R,    KC_T,                        KC_7,    KC_8,    KC_9,    KC_DOT,  KC_BSPC, \
+      KC_LEFT, KC_DOWN, KC_RGHT, KC_F,    KC_G,                        KC_4,    KC_5,    KC_6,    KC_PPLS, KC_PAST, \
+      KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,                        KC_1,    KC_2,    KC_3,    KC_PMNS, KC_PSLS, \
+      KC_LALT, KC_ESC,  KC_TAB,  KC_LSFT, KC_SPC,  KC_LCTL,  ADJUST,   JP_YEN,  KC_0,    KC_COMM, JP_EQL,  KC_ENT   \
+      ),
+
+  /* Adjust (Lower + Raise)
+   * ,----------------------------------.             ,----------------------------------.
+   * | MCR1 | MCR2 | MCR3 | MCR4 | MCR5 |             |DyMcr1|DyMcr2|RcMcr1|RcMcr2|StpRec|
+   * |------+------+------+------+------|             |------+------+------+------+------|
+   * |      |      |      |      |ToGAME|             |      |Qwerty| HDBX |      |      |
+   * |------+------+------+------+------|             |------+------+------+------+------|
+   * |      |      |      |      |      |             |      |      |      |      |      |
+   * |------+------+------+------+------+-------------+------+------+------+------+------|
+   * |RESET |      |      |      |      |      |      |      |      |      |      |      |
+   * `-----------------------------------------------------------------------------------'
+   */
+  [_ADJUST] =  LAYOUT( \
+      MCR1,    MCR2,    MCR3,    MCR4,    MCR5,                      MPLAY1,  MPLAY2,  MREC1,   MREC2,   MRSTOP, \
+      XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, GAME,                      XXXXXXX, QWERTY,  HDBX,    XXXXXXX, XXXXXXX, \
+      XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,                   XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, \
+      RESET,   XXXXXXX, XXXXXXX, XXXXXXX, _______, XXXXXXX, XXXXXXX, _______, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX \
+      )
+};
+
+uint32_t layer_state_set_user(uint32_t state) {
+  state = update_tri_layer_state(state, _RAISE, _LOWER, _ADJUST);
+return state;
+}
+
+bool process_record_user(uint16_t keycode, keyrecord_t *record) {
+  if (!process_record_dynamic_macro(keycode, record)) {
+      return false;
+  }
```

Comment:
This isn't needed anymore, and should be removed.

Reply:
It has changed.